pages/pgql-2.0-spec.md

```diff
@@ -4408,12 +4434,43 @@ The syntax is:
 LateralSubquery ::= 'LATERAL' <Subquery>
 ```
 
+For example, the following query finds the top two people that transacted the most money (largest sum of incoming plus outgoing transfers). For each of those two people, it returns the two largest transactions.
```

Comment:
>largest sum of incoming plus outgoing transfers
The edge is called `transaction` - should we say incoming plus outgoing transactions here?